Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/BearerTokenIdentityResolvers/DefaultBearerTokenIdentityResolverChain.swift

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
This is much like the default AWS credential identity resolver chain. Unlike it tho, this only has one resolver atm.